addons/calendar/static/src/components/activity/activity.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module */
+
+import Activity from '@mail/components/activity/activity';
+import { patch } from 'web.utils';
+import Dialog from 'web.Dialog';
+import core from 'web.core';
+const _t = core._t;
+
+patch(Activity.prototype, 'calendar/static/src/components/activity/activity.js', {
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     * Small override that asks for confirmation in case there is a meeting linked to this activity.
```

Comment:
I don't think it changes much but we usually put the explanation above the various tags

```js
/**
 * blabla
 * 
 * @param xyz
 * @override
 */
```

It was the case in the code you removed, for example.
(Same goes for the other method you added comments to).